jboss-eap-clustered-rhel-VMSS/README.md

```diff
@@ -47,9 +47,9 @@ This template uses a RHEL 8.0 image which is a Pay-as-you-go (PAYG) image and do
 
 2. To deploy the template, you will need:
 
-    -  
+    -  **Admin Username** and password for your VM.
 
-    - *EAP Username** and password to enable the EAP manager UI and Deployment Method.
+    - **EAP Username** and password to enable the EAP manager UI and Deployment Method.
```

Comment:
Modified this line as follows as per the feedback from other PRs for consistency. Please suggest

**JBoss EAP Username** and password